### DISCUSSION TITLE
How validate the data is the same in a role filter?

### DISCUSSION THREAD

**Discussion by Nisgrak** [directus/directus#14258] [Archive]
Hi!

I have a entity with a `fingerprint` field that is set in creation I want that filter locks out the update if didn't send the same fingerprint value, something like this:
![image](https://user-images.githubusercontent.com/19597708/177018761-92850d3c-a9e2-4b9a-9297-1e04c382f9cb.png)

Maybe exist something like that `$ROW`? I only find $CURRENT_USER and $CURRENT_ROLE.

Or maybe have a better way to check this?

Thanks for your help!

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/2993
